spec/aruba/api_spec.rb

```diff
@@ -101,6 +101,26 @@
         expect(File.exist?(File.expand_path(@directory_path))).to be_truthy
       end
     end
+
+    describe '#cd' do
+      context 'with a block given' do
+        it 'runs the passed block in the given directory' do
+          @aruba.create_directory @directory_name
+          full_path = File.expand_path(@directory_path)
+          @aruba.cd @directory_name do
+            expect(Dir.pwd).to eq full_path
+          end
+          expect(Dir.pwd).not_to eq full_path
+        end
+
+        it 'does not touch non-directory environment the passed block' do
```

Comment:
The wording of this `it` seems like it's missing an `of` or a `the` somewhere. Can you try adding words?

Reply:
I updated the wording and added some more specs.

Reply:
That is SUPER! The new wording is _crystal clear_.